api/alembic/versions/304daefe5f80_import_vernon_thessian_polygons.py

```diff
@@ -0,0 +1,28 @@
+"""Import vernon thessian polygons
+
+Revision ID: 304daefe5f80
+Revises: 11b821827824
+Create Date: 2021-11-22 11:30:18.090021
+
+"""
+from alembic import op
+import os
+
+
+# revision identifiers, used by Alembic.
+revision = '304daefe5f80'
+down_revision = '11b821827824'
+branch_labels = None
+depends_on = None
+
+
+file_path = os.getcwd() + '/alembic/vernon_area_prototype/VernonFZBATPolys.shp'
+command = f'ogr2ogr -f "PostgreSQL" PG:"dbname=wps user=postgres" "{file_path}" -nln fire_area_thessian_polygons -append'
```

Comment:
This is probably going to fail - I had to do a `sudo apt install gdal-bin` 1st.

Reply:
You can't have user=postgres - that's not going to work across all platforms. you'll have to read in the username and password from config - also - you won't have that database local, you'll have to read the write db from config ; this may not even be a valid approach at all, because it means we'd need to have gdal-bin installed on all the pods purely for the case of bootstrapping. (so that means re-building the python base image)

I think ogr2ogr actually just neatly wraps a bunch of python that's already imported in our project with osgeo/gdal - so we should be able to do all of this from python script. Just takes some looking into.